pkgs/top-level/all-packages.nix

```diff
@@ -2141,6 +2143,8 @@ in
     gcc = gcc49; # doesn't build with gcc5
   };
 
+  jp = (callPackage ../development/tools/jp { }).bin // { outputs = [ "bin" ]; };
```

Comment:
I see some other derivations doing this but it seems a little backwards to me... Shouldn't "outputs" be in jp/default.nix and not here? I guess it's fine but I don't see why you would override outputs.